make_a_release.sh

```diff
@@ -139,11 +143,11 @@ git commit -a -s -m "Release version $new_ver"
 
 outdir="."
 tag_name="v$new_ver"
-release_name="bmap-tools-$new_ver"
+release_name="bmaptool-$new_ver"
 
 # Create new signed tag
 printf "%s\n" "Signing tag $tag_name"
-git tag -m "$release_name" -s "$tag_name"
+git tag -m "$release_name" "$tag_name"
```

Comment:
The comment and printf do not make sense (if we are dropping the "signing"). In this context, "signing" means adding "Signed-off-by:"  (`-s`) I believe.

Reply:
In this case ("git tag") the signing refers to actually signing the tag with a public key of some sort, not the "adding the SoB line" thing. This is new to me, I don't have a public key setup anywhere, so I want to drop it so I can make releases. If it's something that I need to add I can look into it later and add it back.

I'll sync the action with the printf comments.

Reply:
```
$ man 1 git-tag
...
       -s, --sign
           Make a GPG-signed tag, using the default e-mail address’s key. The default behavior of
           tag GPG-signing is controlled by tag.gpgSign configuration variable if it exists, or
           disabled otherwise. See git-config(1).
```